docs/outposts.md

```diff
@@ -0,0 +1,51 @@
+# Using EBS CSI Driver on AWS Outposts
+
+## Overview
+
+When using the EBS CSI Driver on AWS Outposts, there are several important considerations to be aware of.
+
+### IMDS Metadata Access
+The EBS CSI driver requires access to Instance Metadata Service (IMDS) to function properly on Outposts. Ensure that:
+- IMDS is not disabled on your EC2 instances
+- If using IMDSv2 ensure the hop limit is set to 2 
+
+### First Generation Outpost Racks
+- You must explicitly specify a supported volume type (e.g., `gp2`). Check AWS documentation for the latest supported EBS volume types on your specific Outpost configuration.
```

Comment:
np: `Check AWS documentation` -> `Check the AWS documentation`

Also, maybe provide a link?

Reply:
Also also, see my other comment but basically don't say "first generation outposts" anywhere here.